pennylane/hf/molecule.py

```diff
@@ -110,3 +110,89 @@ def __init__(
         self.nuclear_charges = [atomic_numbers[s] for s in self.symbols]
 
         self.n_electrons = sum(np.array(self.nuclear_charges)) - self.charge
+
+        self.mo_coefficients = None
+
+    def atomic_orbital(self, index):
+        r"""Return a function that evaluates a basis function at a given position.
+
+        Args:
+            index (int): index of the basis function
```

Comment:
Is there a short way to explain how the indices are ordered? I assume that lower index means lower energy, right? 
It may just not be so clear to users what the indices represent exactly

Reply:
The indices are ordered according to the order of element symbols.